### PR TITLE
Add TO Go cachegroups/id/queue_update

### PR DESCRIPTION
**Pull request by rob05c**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1619/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1642/
Test PASSed.

File: traffic_ops/traffic_ops_golang/cachegroup/queueupdate.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/lib/go-tc"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/auth"
+)
+
+type QueueUpdatesReq struct {
+	Action string      `json:"action"`
+	CDN    *tc.CDNName `json:"cdn"`
+	CDNID  *string     `json:"cdnId"`
```
Comment (DylanVolz) [inline traffic_ops/traffic_ops_golang/cachegroup/queueupdate.go]: On the perl side an int or string works, I think int is the type we want moving forward, with support for a string for backwards compatibility.

**Comment by DylanVolz** [comment]
Tested, this works as expected other than the noted difference that per supports both an int or string as the cdnId in the request body.

File: traffic_ops/traffic_ops_golang/cachegroup/queueupdate.go (hunk 1 above)
Comment (rob05c) [reply traffic_ops/traffic_ops_golang/cachegroup/queueupdate.go]: I agree int is better, but it's documented as being a string.

Normally I don't like making the API support undocumented things, it leads to difficult to support, maintain, and understand clients and servers. But in this case, it seems likely many clients are passing the ID as an int.

 I'll change it to decode either.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1770/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1771/
Test PASSed.

**Comment by rob05c** [comment]
>Tested, this works as expected other than the noted difference that perl supports both an int or string as the cdnId in the request body.
Changed

**Comment by DylanVolz** [comment]
Retested, still works as expected and accepts string or int as id in body.

**Pull request merged**